src/olympia/addons/migrations/0049_clear_bad_url_data.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 4.2.7 on 2023-12-06 15:17
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('addons', '0048_deletedpreviewfile_alter_addonuser_original_role_and_more'),
+        ('translations', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.RunSQL(
```

Comment:
You manually tested this (I hope)?

Reply:
Yes. You can verify this by manually creating bad `homepage`/`support_url` data in a shell and running the migration.